tests/integration/test_aws/test_remove_from_bucket.py

```diff
@@ -91,6 +91,9 @@ def test_remove_from_bucket(
         - The `configuration_defaults` file provides the module configuration for this test.
         - The `cases_defaults` file provides the test cases.
     """
+    if metadata['name'] in ['alb_remove_from_bucket', 'clb_remove_from_bucket', 'nlb_remove_from_bucket']:
+        pytest.skip(reason='ALB, CLB and NLB integrations are removing older logs from other region')
```

Comment:
Could you explain this part? I think I don't quite understand it.

Reply:
I moved to another location in 5e31c38c54d8a9a5bd04e9b07b1802523a597dfa.
I found a bug, reported in https://github.com/wazuh/wazuh/issues/15804. These tests delete files that other tests need and make them fails.
Is for this reason that I choose to skip until we fix the problem.